### ISSUE TITLE
Scaffolding db with model output to another folder messes up namespaces

### ISSUE THREAD

**Issue by divinebovine** [dotnet/efcore#18603]
When scaffolding an existing database and using the the output-dir option, the context is missing a needed using directive.

Here is the command that I'm using to scaffold our database:
```sh
dotnet ef --verbose dbcontext scaffold Name=SmplDatabase Microsoft.EntityFrameworkCore.SqlServer --force --startup-project ../EfCoreStartup/EfCoreStartup.csproj --data-annotations -c SMPLContext --context-dir . -o Models
```

Our database models are correctly going to the Models directory. They are given a namespace that matches the root namespace with `.Models` added to the end. Howver, the SMPLContext.cs file is missing a using statement for that namespace.

### Further technical details

EF Core version: 3.0
Database provider: Microsoft.EntityFrameworkCore.SqlServer
Target framework: netstandard 2.1 and netstandard 2.0
Operating system: Windows
IDE: vscode 1.39.2

**Comment by ErikEJ** [comment]
This is fixed in 3.1, I beleive

**Comment by ajcvickers** [comment]
Duplicate of #17839
